### ISSUE TITLE
WARNING_TRANSCRIPT_NO_START_CODON

### ISSUE THREAD

**Issue by AroobAlhumaidy** [pcingola/SnpEff#275]
Hello, the ERROR_CHROMOSOME_NOT_FOUND issue is solved thanks to you, 

when the chromosome name issue fixed another error showed up, which is:

WARNINGS: Some warning were detected
Warning type    Number of warnings
WARNING_TRANSCRIPT_NO_START_CODON       118
WARNING_TRANSCRIPT_NO_STOP_CODON        10

Thanks, I appreciate your help,

**Comment by chenqing-1996** [comment]
hello，AroobAlhumaidy
I have a problem "ERROR_CHROMOSOME_NOT_FOUND",how did you fix it?

**Comment by AroobAlhumaidy** [comment]
Hello chenqing-1996 , 
i got this solution page from Pablo Cingolani, Here you go: https://github.com/pcingola/SnpEff/wiki/ERROR_CHROMOSOME_NOT_FOUND 

best of luck

**Comment by pcingola** [comment]
Those are warnings that might appear on some poorly annotated genes.

Here is a description of the error messages: https://github.com/pcingola/SnpEff/wiki/Error-and-Warning-messages

**Comment by chenqing-1996** [comment]
> Hello chenqing-1996 ,
> i got this solution page from Pablo Cingolani, Here you go: https://github.com/pcingola/SnpEff/wiki/ERROR_CHROMOSOME_NOT_FOUND
> 
> best of luck

AroobAlhumaidy,thank you so much.I have read this reply, but I downloaded the snpEff database directly, there is no input file for reference, and I have tried to build the database by myself before, but that was also using fasta file. How should I deal with this situation?

**Comment by chenqing-1996** [comment]
> Hello chenqing-1996 ,
> i got this solution page from Pablo Cingolani, Here you go: https://github.com/pcingola/SnpEff/wiki/ERROR_CHROMOSOME_NOT_FOUND
> 
> best of luck

AroobAlhumaidy,thank you so much.I have read this reply，I have a question , I got # Chromosomes names [sizes]  : '1' [249250621] '2' [243199373],Ishould adjust my chromosome name as '1','2' or 1,2

**Comment by chenqing-1996** [comment]
> Those are warnings that might appear on some poorly annotated genes.
> 
> Here is a description of the error messages: https://github.com/pcingola/SnpEff/wiki/Error-and-Warning-messages

Hello pcingola，
thank you for your reply.I was a little confused with snpEff. If it annotated according to the size of each chromosome, can't genomes assembled into Contigo or scaffol be able to annotate?Although I have thought about making a pseudochromosome , it is not accurate .Is it meaningless to make annotation like this by using snpEff?
Looking forward to your reply!

**Comment by pcingola** [comment]
Yes, "chromosome" could also be a contig, or a scaffold (I generically called them all "chromosomes").
